docs/source/index.rst

```diff
@@ -192,6 +192,7 @@ Deployment
     getting-started/index
     new-features/index
     llms/index
+    llms/rag/index
```

Comment:
We definitely don't want to add this here. This is the top-level. You can pull the changes from my modification to your PR here: https://github.com/BenWilson2/mlflow/tree/update-docs 
Feel free to remote add, fetch, then merge the changes to get the navigation elements and notebook view / download linkages incorporated into your PR :D

Reply:
Or if you'd like to just see what I changed... https://github.com/mlflow/mlflow/compare/docs-refactor...BenWilson2:mlflow:update-docs?expand=1
Whichever works for you!